wpilibc/src/main/native/cpp/smartdashboard/SmartDashboard.cpp

```diff
@@ -29,11 +29,23 @@ struct Instance {
 };
 }  // namespace
 
-static Instance& GetInstance() {
-  static Instance instance;
+static std::unique_ptr<Instance>& GetInstanceHolder() {
+  static std::unique_ptr<Instance> instance = std::make_unique<Instance>();
```

Comment:
Using this approach with unique_ptr probably adds a tiny bit of overhead because it's going to end up on the heap somewhere... but I think most of the global instances are either not called very often (or likely just at robot initialization time) or the added overhead will be tiny compared to whatever they do.